README.md

```diff
@@ -66,6 +66,35 @@ page =
   |> MyApp.Repo.paginate(page: 2, page_size: 5)
 ```
 
+## Supported Options
+
+There are several options that can be supplied in a dedicated `options` keyword list arg. Here is an example of using `options`...
+
+```elixir
+defmodule MyApp.Repo do
+  use Ecto.Repo, otp_app: :my_app, adapter: Ecto.Adapters.Postgres
+  use Scrivener, options: [allow_overflow_page_number: true]
+end
+```
+
+Here is an explanation of the supported options...
+
+* `:allow_overflow_page_number` - If set to true then when you supply a page greater than the total pages, it will return an empty list for entries. If set to false then it will (silently) treat any supplied `page` arg as if it was equal to the total pages. Default is false.
```

Comment:
On line 71 you mention "there are several supported options", but then we proceed to only document a single option.   We should either a) document the options available that scrivener_ecto uses, or b) point people (via link) to the documented options directly in the Scrivener docs.